doc/source/examples/analysis/structure/elastic_network.ipynb

```diff
@@ -77,7 +77,15 @@
      "shell.execute_reply": "2021-05-19T05:58:11.919074Z"
```

Comment:
<p>I'm not seeing any changes on how this is being run, do we have a reason why the results here are drastically different? (did the core library change in some way?)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MDAnalysis/UserGuide/pull/241/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>good catch. It's just that <code>results</code> has a different meaning now, so that length is the number of keys in that dictionary (eigenvalues + eigenvectors + times).</p><p>Fixed. </p>